src/gui/generalsettings.cpp

```diff
@@ -30,21 +30,107 @@
 
 #include "ignorelisteditor.h"
 #include "common/utility.h"
+#include "logger.h"
 
 #include "config.h"
 
 #include "legalnotice.h"
 
+#include <QFileDialog>
+#include <QMessageBox>
 #include <QNetworkProxy>
 #include <QDir>
 #include <QScopedValueRollback>
 
+#include <private/qzipwriter_p.h>
```

Comment:
Why must the interesting parts always be `private/`? 😁

However, luckily it's not going to change soon ( https://bugreports.qt.io/browse/QTBUG-3897 ).
And still in the same place with Qt 5.15.0

Reply:
Yep, I've known this class for years. It was this or pulling KArchive but didn't want to do this right now.

We might need to reevaluate for Qt 6, but so far it's still there in the dev branch and didn't change.